src/aind_metadata_service/models.py

```diff
@@ -33,3 +34,9 @@ class FundingInformation(Funding):
     information from the Funding SmartSheet"""
 
     investigators: Optional[str] = Field(default=None)
+
+
+class ViralMaterialInformation(ViralMaterial):
+    """Viral Material with Stock Titer from SLIMS"""
+
+    stock_titer: Optional[List[int]] = Field(default=None)
```

Comment:
why is the stock titer a List?

Reply:
It's a list because in TARS, a user can technically add more than one Titer object.

Reply:
If we have a list of titers, we need to know what they map to. A single plasmid can only have one titer, so then we need a matching list for the plasmids. 

It doesn’t make sense to have a “preferred” titer vs non-preferred, so that suggests to me that some of this is TARS functionality that they are just duplicating between modules. But if multiple titers is a thing for a prep lot, then we have to map plasmids differently than we do.

Reply:
I can make the plasmid_tars_alias a list in the TARS endpoint response for now. David suggested we make it a list in the data-schema in a later ticket (Union[str, List]).

Reply:
I think we need to do it right and have the plasmid being it’s own class. 
But right now looking at LIMS I don’t see a link between plasmids and titer, which means nobody has the information. Can we find an actual example of a prep lot with multiple plasmids. We’re doing all this work to accomodate what I understand is an uncommon (but real) use case and it would really help to have an actual example to work with rather than trying to build for hypotheticals.

Reply:
I talked with Aaron. (a) no examples exist in TARS and (b) there will only be one titer because the plasmids are packed in one capsid. So make titer not a list and we’re good to go.

Reply:
There will probably be only 1 preferred molecule alias so I think there will only ever be one plasmid_tars_alias. I need to search for an example.

Reply:
Aight update from Aaron: "as of right now there should only be one titer. We are only tracking the titer of the virus before it is aliquoted out into vials. The long term plan is to track each of the aliquots titers which may be different. " I'll update this PR to not be a list.